lua/gatekeeperc.lua

```diff
@@ -0,0 +1,78 @@
+local ffi = require("ffi")
+
+--Structs
+ffi.cdef[[
+
+struct bp_config {
```

Comment:
Can't we use Cody's suggestion (http://lua-users.org/lists/lua-l/2011-02/msg01485.html)?